packages/web/src/js/__tests__/Modal.test.ts

```diff
@@ -99,4 +99,22 @@ describe('Modal', () => {
       expect(Modal.getOrCreateInstance(dialog)).toBeInstanceOf(Modal);
     });
   });
+
+  describe('backdropCloseDisabled', () => {
```

Comment:
```suggestion
  describe('backdrop', () => {
```

And also add the opposite test.

Reply:
changed